content/en/service_management/workflows/datastore.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Apps Datastore
+disable_toc: false
+further_reading:
+- link: "service_management/app_builder/build"
+  tag: "Documentation"
+  text: "Build Apps"
+- link: "service_management/workflows/build"
+  tag: "Documentation"
+  text: "Build Workflows"
+---
+
+{{< callout url="https://docs.google.com/forms/d/1gVVWLg5P3EXkVXBc4qZtqKp_nqykDsFXCMmh6pbvxx0/edit" btn_hidden="false" header="Try the Preview!">}}
+Datastore is in Preview. Use this form to request access today.
+{{< /callout >}}
+
+## Overview
```

Comment:
- I think we should emphasis that you can interact with the datastore from an app or from a WF. Currently, we only say: "structured data storage solution within Datadog's App Builder and Workflow Automation products" Do you think it is clear enough?
- It is very important to mention that the Datastore has also a UI layer that allow users to interact with the table from the UI. This is our advantage over solutions like S3 or DynamoDB

Reply:
Added some things to the intro